WooCommerce/Classes/ViewRelated/Dashboard/DashboardViewModel.swift

```diff
@@ -431,6 +485,15 @@ private extension DashboardViewModel {
         }
     }
 
+    @MainActor
+    func syncOrders() {
+        let action = OrderAction.synchronizeOrders(siteID: self.siteID,
+                                               statuses: nil,
+                                               pageNumber: Constants.orderPageNumber,
+                                               pageSize: Constants.orderPageSize) {_, _ in }
```

Comment:
Super nit: The indent should be updated.

Reply:
You're right, added in https://github.com/woocommerce/woocommerce-ios/pull/12521/commits/18a5542dc98e277a00c2b2f9bfe4503aaefcd233